skin-schema.json

```diff
@@ -716,6 +716,36 @@
         }
       }
     },
+    "isVrAnimationEnabled": {
+      "description": "Shows if vr inform animation is enabled",
+      "type": "object",
+      "properties": {
+        "vrNotification": {
+          "description": "Vr notification should be shown",
+          "type": "boolean",
+          "default": "If no value is provided for vrNotification, vrNotification will default to true."
+        },
+        "vrIcon": {
+          "description": "Vr icon should be  shown",
+          "type": "boolean",
+          "default": "If no value is provided for vrIcon, vrIcon will default to false."
+        }
+      }
+    },
+    "animationDurations": {
+      "description": "Sets duration for vr animation",
```

Comment:
Should this be vrAnimation? In other places, like 3 lines below, I see "vrNotification." Let's shoot for consistency in our docs

Reply:
\+